bundles/org.openhab.binding.meross/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="meross" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Meross Binding</name>
+	<description>This is the binding for Meross.</description>
```

Comment:
Would be nice if you can extend the description a little bit?

Reply:
ok I changed to "Binding for Meross Internet Of Things environment" would it be ok?